src/GrumPHP/Console/Application.php

```diff
@@ -116,6 +116,10 @@ protected function getDefaultCommands()
             $container->get('config'),
             $container->get('locator.changed_files')
         );
+        $commands[] = new Command\Git\PrePushCommand(
+            $container->get('config'),
+            $container->get('locator.changed_files')
```

Comment:
Also here we'll need the files that are being pushed to the remote instead of the staged files.